module_ctx.go

```diff
@@ -154,6 +155,9 @@ type ModuleContext interface {
 	BaseModuleContext
 
 	OtherModuleName(m Module) string
+	OtherModuleDir(m Module) string
```

Comment:
1. Would it be better to have a single function returning struct than five OtherModuleXxxx functions with same arguments?
2. Please document what these functions do.

Reply:
1. I don't think so, most of the callers only want a single value.
2.  I'll do that in a follow-up, the whole interfaces need to be documented, not just these new functions.